### PR TITLE
WL-1253 Migrate to GitHub Actions

### PR DESCRIPTION
**Pull request by jadolg**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+on:
+  push:
+    branches: [ master ]
+    tags:
+      - 'v*'
+
+jobs:
+
+  build:
+    if: startsWith(github.ref, 'refs/tags/v')
```
Comment (aborilov) [inline .github/workflows/ci.yml]: do we need to have this check together with `tags: - 'v*'` ?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+name: ci
+
+on:
+  push:
+    branches: [ master ]
```
Comment (hvoigt) [inline .github/workflows/ci.yml]: Similarly, as below should we solely trigger on tags?

```suggestion
```

**Review by hvoigt — COMMENTED** [review]
Almost good.

File: .github/workflows/ci.yml (hunk 2 above)
Comment (hvoigt) [reply .github/workflows/ci.yml]: @jadolg here the master branch is still in the PR right? Or am I looking wrong?

**Review by hvoigt — APPROVED** [review]
LGTM, but please remove the `master` branch from the triggers. As this is old code it probably does not matter so much (just wasting CI time) that is why I am approving this now.

File: .github/workflows/ci.yml (hunk 2 above)
Comment (jadolg) [reply .github/workflows/ci.yml]: fixed in [6b72385](https://github.com/Jimdo/yaml-vault/pull/3/commits/6b723853f7c81121db551305d600c7db544d5c88)

File: .github/workflows/ci.yml (hunk 1 above)
Comment (jadolg) [reply .github/workflows/ci.yml]: fixed in [6b72385](https://github.com/Jimdo/yaml-vault/pull/3/commits/6b723853f7c81121db551305d600c7db544d5c88)

**Pull request merged**